.github/workflows/test-e2e-pr.yaml

```diff
@@ -1,4 +1,4 @@
-name: E2E Tests
+name: E2E PR Tests
```

Comment:
Updated the name here because we had 2 workflow files with the same `name:` value